modules/data.land/R/ic_process.R

```diff
@@ -50,6 +50,20 @@ ic_process <- function(settings, input, dir, overwrite = FALSE){
   con <- bety$con
   on.exit(db.close(con))
   
+  
+  # setup site database number, lat, lon and name and copy for format.vars if new input
+  new.site <- data.frame(id = as.numeric(site$id), 
+                         lat = PEcAn.data.atmosphere::db.site.lat.lon(site$id, con = con)$lat, 
+                         lon = PEcAn.data.atmosphere::db.site.lat.lon(site$id, con = con)$lon)
```

Comment:
Seems silly to query the database twice to get lat/lon when the function returns both